es/news/_posts/2016-02-24-support-plan-of-ruby-2-0-0-and-2-1.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: news_post
+title: "Planes de soporte para Ruby 2.0.0 y Ruby 2.1"
+author: "usa"
+translator: 'Fernando Perales'
+date: 2016-02-24 09:00:00 +0000
+lang: es
+---
+
+Anunciamos los planes de soporte a futuro para Ruby 2.0.0 y Ruby 2.1.
+
+## Sobre Ruby 2.0.0.
+
+Como se anunció con anterioridad, todo el soporte para Ruby 2.0.0.
```

Comment:
Please, update this: `Como se anunció con anterioridad, todo el soporte para Ruby 2.0.0.` to `Como se anunció anteriormente, todo el soporte para Ruby 2.0.0`.